src/main/java/com/neuvector/Scanner.java

```diff
@@ -329,6 +332,30 @@ private static String maskCredential(String message, String credential){
         return message.replace(credential, "******");
     }
 
+    private static String getDockerUserAndGroup() {
+        String user = executeBashCommand("id -u");
+        String group = executeBashCommand("id -g");
```

Comment:
I'm not sure I understand where this code runs, so maybe my concern here is not important. But if it runs on Windows, the above command will fail. @koraytugay WDYT?

Reply:
Container scanning does not support Windows. Only mac/linux..

Reply:
@eduard-tita This code runs in insight-scanner.. Before we start the neuvector container.. For docker-nexus-iq-cli, it would run within the docker-nexus-iq-cli..

Reply:
@koraytugay What if you run container scanning with the Java IQ CLI on Windows? Isn't the IQ CLI runs the insight-scanner, so indirectly it will run the above code on Windows?

Reply:
@eduard-tita Yes, it will try to run it. Even if you run it with nexus-iq-cli.jar or so.. I do not know what the consequences will be, probably it will just throw some sort of an IO exception? What I meant was - this is not supposed to work in Windows (because the neuvector container does not support windows) - but you are right. People can still try it..

Reply:
According to this https://stackoverflow.com/questions/59716461/win32-equivalent-of-getgid/59716588#59716588 Windows doesn’t have a concept of a single effective group ID

Reply:
I know that. That's why I asked.  :smiley:

Reply:
According with what Koray mentioned `Container scanning does not support Windows. Only mac/linux..` I think we don't have to be aware of this, are you agree?